src/main/java/net/sacredlabyrinth/phaed/simpleclans/ClanPlayer.java

```diff
@@ -826,8 +826,7 @@ public String getRankDisplayName() {
     @Placeholder("has_rank")
     public boolean hasRank() {
         if (clan != null) {
-            Rank r = clan.getRank(getRankId());
-            return r != null;
+            return clan.getRank(getRankId()) != null;
         }
         return false;
```

Comment:
`return clan != null ? clan.hasRank(getRankId()) : false`
or even shorter:
`return !getRankId().isEmpty();`

Reply:
Second case doesn't take care of accidentual situations, 
where incorrect rank is set somehow, but I still prefer it over the first one.